src/advanced/network.md

```diff
@@ -58,6 +58,24 @@ full node has been really authorized by a supermajority of validators.
 Full nodes use the
 [Exonum binary serialization format](../glossary.md#binary-serialization)
 over TCP to communicate with each other.
+
+Starting with Exonum 0.8, all network connections are encrypted using
+[Noise Protocol][noise]. The Protocol starts with a handshake message exchange
+when the parties exchange [Diffie-Hellman (DH) public keys][DH], perform a
+series of DH operations and, as a result, receive a shared secret key. This
+key is then used to send encrypted messages.
+
+Noise Protocol protects Exonum against a number of potential vulnerabilities,
+for example, traffic sniffing between nodes.
+
+!!! warning
+    Nodes compiled with previous versions of Exonum will not connect to nodes
+    updated to 0.8 and further.
+
+!!! note
+    The current implementation of connection encryption is not complete: the
+    Noise channels are encrypted, but not authenticated.
```

Comment:
Since version 0.9 connections are authenticated. We need to specify a remote public key when establishing a connection and receiver node will authenticate sender via that key.